ugo_collector/RatePlan_Created/RatePlan_Created_1_firstAnalysis.ipynb

```diff
@@ -0,0 +1,2472 @@
+      "text/plain": [
+       "<Figure size 1296x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(18,8))\n",
+    "sns.set(font_scale = 1.5)\n",
+    "sns.countplot(x = OTA_HotelRateAmountNotifRQ.Name, hue = OTA_HotelRateAmountNotifRQ.bidroom_hotel_id, data = OTA_HotelRateAmountNotifRQ, palette = 'viridis')\n",
+    "plt.title('Breaking Name by Origin')\n",
+    "plt.show()"
```

Comment:
Title repeated from plot above, should be `Breaking Name by bidroom_hotel_id`

Reply:
@bluejurand thank you! you are rght